lib/src/navigation/router.dart

```diff
@@ -1,116 +1,116 @@
-// GoRouter configuration
```

Comment:
Can we remove this file, if it's not needed?

@divyanshgandhilm to solve the navigation problem and attain parity with the navigation flow that we have in Feed we removed go router.

For notification and share post navigation, we'll be using navigator key to navigate to the respective screen